I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 21, in script
    k "Командир русской гвардией: Братья мои,внимайте же меня все в сей момент!
  File "renpy/common/000window.rpy", line 98, in _window_auto_callback
    _window_show()
  File "renpy/common/000window.rpy", line 60, in _window_show
    renpy.with_statement(trans)
IOError: Couldn't find file 'as.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 21, in script
    k "Командир русской гвардией: Братья мои,внимайте же меня все в сей момент!
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\ast.py", line 671, in execute
    statement_name("say")
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\ast.py", line 45, in statement_name
    i(name)
  File "renpy/common/000window.rpy", line 98, in _window_auto_callback
    _window_show()
  File "renpy/common/000window.rpy", line 60, in _window_show
    renpy.with_statement(trans)
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\exports.py", line 1601, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\display\core.py", line 2251, in do_with
    clear=clear)
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\display\core.py", line 2702, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\display\core.py", line 3380, in interact_core
    renpy.audio.audio.periodic()
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\audio\audio.py", line 1060, in periodic
    six.reraise(exc[0], exc[1], exc[2])
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\audio\audio.py", line 1038, in periodic_thread_main
    periodic_pass()
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\audio\audio.py", line 979, in periodic_pass
    c.periodic()
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\audio\audio.py", line 457, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\audio\audio.py", line 73, in load
    rv = renpy.loader.load(fn)
  File "D:\Games\Рен пай\renpy-7.3.5-sdk\renpy\loader.py", line 576, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'as.mp3'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
dfig 1.0
Thu Dec 17 20:31:26 2020
